backend/ohq/statistics.py

```diff
@@ -232,3 +234,83 @@ def queue_calculate_questions_per_ta_heatmap(queue, weekday, hour):
         hour=hour,
         defaults={"value": statistic if statistic else 0},
     )
+
+
+def user_calculate_questions_asked(user):
+    num_questions = Question.objects.filter(asked_by=user).count()
+
+    if num_questions and not num_questions == 0:
```

Comment:
Could be wrong, but for these checks I think you can just write `if num_questions:`. If `num_questions` is ever 0, the if statement would fail (0 is considered `False`)